htsinfer/htsinfer.py

```diff
@@ -85,15 +85,15 @@ def evaluate(self):
                 self.get_library_stats()
                 LOGGER.info(
                     "Library stats determined: "
-                    f"{self.config.results.library_stats.json()}"
+                    f"{self.config.results.library_stats.model_dump_json()}"
```

Comment:
These changes should not show up here. Please merge the updated default branch that already has these changes from the other PR into your feature branch.

Reply:
I removed it and will create a separate PR for these changes.